src/Support/Support/ChangeLog.md

```diff
@@ -20,6 +20,9 @@
 
 ## Upcoming Release
 
+## Version 1.0.1
```

Comment:
please do not change version, Azure-PowerShell team will bump it during each release